admin/console/other-useful-commands.rst

```diff
@@ -18,8 +18,8 @@ Reprocess unprocessable mails
 
 When Zammad encounters a mail it cannot parse (e.g. due to a parser bug or a
 malformed message), it will store the mail in
-``tmp/unprocessable_mail/<ID>.eml``,  give up on attempting to parse the mail,
-and will warn on the monitoring page that there are unprocessed mails.
+``var/spool/unprocessible_mail/<ID>.eml``,  give up on attempting to parse the
```

Comment:
Please correct this to `var/spool/unprocessable_mail`, that's what the code uses.

Reply:
:see_no_evil: Ooops. Thank you for your hint!!